pom.xml

```diff
@@ -892,14 +901,14 @@
                   <version>2.10.4</version>
               </plugin>
               <plugin>
-                  <groupId>org.codehaus.mojo</groupId>
-                  <artifactId>findbugs-maven-plugin</artifactId>
-                  <version>${findbugs-maven-plugin.version}</version>
+                  <groupId>com.github.spotbugs</groupId>
+                  <artifactId>spotbugs-maven-plugin</artifactId>
+                  <version>${spotbugs-maven-plugin.version}</version>
```

Comment:
Is spotbugs a fork, or did fb just change ownership / rename

Reply:
fb retired, and spotbugs is the new fork